internal/controller/apiextensions/claim/reconciler_test.go

```diff
@@ -791,7 +793,7 @@ func TestReconcile(t *testing.T) {
 
 			got, err := r.Reconcile(context.Background(), reconcile.Request{})
 
-			if diff := cmp.Diff(tc.want.claim, tc.args.claim, test.EquateErrors()); diff != "" {
+			if diff := cmp.Diff(tc.want.claim, tc.args.claim, cmpopts.EquateApproxTime(30*time.Second)); diff != "" {
```

Comment:
Perhaps a comment explaining why we equate time approximately would be useful for future readers?

Also, I don't feel strongly but 30 secs feels like a long time. I've seen this twice now and each time it was a 1 second difference. I went with 3 seconds.

Reply:
Nevermind, I dropped these comments slightly too late and they're nothing serious so feel free to ignore.